client/landing/stepper/declarative-flow/link-in-bio.ts

```diff
@@ -40,7 +40,7 @@ export const linkInBio: Flow = {
 						return navigate( 'patterns' );
 					}
 					return window.location.replace(
-						'/start/account?flowName=link-in-bio&redirect_to=/setup/patterns?flow=link-in-bio'
+						'/start/account/user?variationName=link-in-bio&pageTitle=Link%20in%20Bio&redirect_to=/setup/patterns?flow=link-in-bio'
```

Comment:
The page title is a bit risky to parameterize, if someone changes it, the UI will reflect it and they'll make a wordpress.com link that renders whatever they want to say (imagine something offensive).